tests/cli-configuration.spec.js

```diff
@@ -265,4 +266,14 @@ describe('CLI Configuration', function() {
 
   });
 
+  describe('.load()', function() {
+
+    it(`should load defualt internal ${packageJSON.name}.conf.js if file is not found`, function() {
```

Comment:
Yeah, I agree. Lets bail out if the user specifies a configuration file but it doesn't exist.